client/packages/android/README.MD

```diff
@@ -42,11 +63,38 @@ yarn apply-config
 ```
 
 As per earlier comment, please make sure to build front end with `yarn build` from client directory prior to runner the above or run `yarn android:{command here}` from client directory to do both with one command.
+For this reason it's often easier to run the following scripts in the `client` directory which makes sure everything is built correctly:
+
+```bash
+# in the client directory:
+
+# Build the frontend and the server libs
+yarn android:build:server
+# Build the frontend and assembles an Android debug apk
+yarn android:build:debug
+# Build the frontend and assembles an Android release apk
+yarn android:release
+```
 
 ### Know script issues
 
 When building apk `properties.local` files is required (it can be empty). Also can sometime get a lock on gradle build, can run `find ~/.gradle -type f -name "*.lock" -delete` to fix it
 
+## Debugging
+
+It's recommended to use Android Studio for debugging and running Android App, there is a lot of functionality that cannot be replicated in Visual Studio Code (although there are some extensions for Visual Studio Code they are quite limited).
```

Comment:
```suggestion
It's recommended to use Android Studio for debugging and running Android App, there is a lot of functionality that cannot be replicated in Visual Studio Code (although there are some extensions for Visual Studio, Code they are quite limited).
```

Reply:
Comma doesn't make sense there...

Reply:
Oh sorry meant to move it after. Visual Studio Code, they are quite limited XD